### ISSUE TITLE
Specify directory path for native dlls

### ISSUE THREAD

**Issue by amits1995** [oysteinkrog/SQLite.Net-PCL#95]
When I used the library from an asp.net site that I developed, SQLitePlatformWin32 threw an exception because it couldn't load the native dll. this is happening because when you debug your asp.net site, windows uses a temp path to hold your site's assemblies. if I could just specify it where it can find the native dll's the problem will be solved.

**Comment by oysteinkrog** [comment]
Hi, take a look at the following method:
static SQLiteApiWin32Internal()
This is where I do the LoadLibrary magic to preload the sqlite dll, it's a bit tricky to make this configurable since it is a static ctor right now, but I suppose I could change it to be instanced and initialized through the platform.
It does seem a bit like a hack though, why is content not copied when running tests?

**Comment by amits1995** [comment]
Hi, as I already said, when I debugged my asp.net application I realized that microsoft did a strange thing and put every dll of the project in a different temp folder and the sqlite dll was alone in a folder with no x86 or x64 folders around it. Of course I tried forcing it to copy those folders local with no success. I did build a dll of my own that does set some static property so I could specify where the dlls are and it seems like a legit feature to me. Of course the static property is not a good practice solution. Look, I think that for some people it can be useful. It was critical for me. An optional parameter can be nice... But it's not a must-have change. Best of luck!
